app/scripts/setup.sh

```diff
@@ -74,18 +74,24 @@ function createSymlinks() {
     esac
 }
 
-function checkoutSilme() {
-    # Check out SILME library to a specific version (0.8.0)
-    if [ ! -d $libraries/silme/.hg ]
+function setupExternalLibraries() {
+    # Check out or update compare-locales library
+    version="RELEASE_1_2_1"
```

Comment:
One more note: I though about that, but I couldn't come up with an acceptable solution.

We could checkout tmx_products as an external library, like we do with p12n-extract, but that wouldn't really solve the issue with compare-locales (I can't run hg pull every time the library it's opened, only clone the repo if it's missing).

The alternative would be to install compare-locales as a proper Python package (changing the path is frowned upon), and maybe use a virtual env. It would be much cleaner, but it would also introduce more dependencies on the server.